### PR TITLE
Editor: Layer panel redesign

### LINKED ISSUE #6210: Redesign: Layer panel
## Feature Description

Update design for layer panel based on [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3360%3A63235).

- [ ] Update overall size and shape (notice how the layer panel is narrower than the full inspector and has rounded corners on top)
- [ ] Add lock icon for background using design system
- [ ] Make sure layers are correctly laid out and spaced

Do not update the panel header including title, arrow and border in this PR.

| Current design | New design |
|-|-|
| Screen Shot 2021-02-04 at 23 46 35 | Screen Shot 2021-02-04 at 23 49 19 |
| Screen Shot 2021-02-04 at 23 46 31 | Screen Shot 2021-02-04 at 23 49 47 |

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by samwhale**
## Context

The layer panel was redesigned: [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3360%3A63235).

## Summary

Updates the layer panel to match the new designs.

## Relevant Technical Choices

- `LayerIcon` and `LayerContent` are only used in the layer panel, so editing those shouldn't break anything in the editor.
- I tried to keep everything consistent so I am sharing a `<LayerText />` component for the text content of each row.

## To-do

nothing

## User-facing changes

|Before|After|
|--|--|
|image|new-layer-panel|

## Testing Instructions

1. Go to editor
2. Add one of each element type to the canvas and check out the layers in the layer panel

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1. Go to editor
2. Add one of each element type to the canvas and check out the layers in the layer panel

### UAT

- [x] UAT should use the same steps as above.

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6210

### CODE REVIEW

File: assets/src/edit-story/elements/gif/icon.js [hunk 1]
```diff
@@ -14,19 +14,25 @@
...
  */
-import { Media } from '../../icons';
+import StoryPropTypes from '../../types';
+import VisibleImage from '../media/visibleImage';
+import { getSmallestUrlForWidth } from '../media/util';
 
-function GifIcon() {
-  const alt = __('GIF element', 'web-stories');
-  return <Media aria-label={alt} />;
+function GifLayerIcon({
```
Comment (samwhale) [inline assets/src/edit-story/elements/gif/icon.js:24]: For all of these I did the following:
- Renamed the `layer.js` to `icon.js` since the `LayerContent` is now the icon in the new designs
- Renamed `icon.js` to `layer.js`
- Edited the new `layer.js` to have the necessary text content

The diffs are confusing, sorry about that! I tried to keep with the existing file structure.

File: assets/src/edit-story/elements/gif/layer.js [hunk 2]
```diff
@@ -13,25 +13,28 @@
...
-}) {
-  const src = getSmallestUrlForWidth(0, resource);
-  return <VisibleImage src={src} alt={alt} height="20" />;
-}
+function GifLayerContent({ isBackground, element }) {
+  const { alt } = element?.resource || {};
 
+  return (
+    <LayerText isBackground={isBackground}>
+      {alt || __('GIF', 'web-stories')}
```
Comment (samwhale) [inline assets/src/edit-story/elements/gif/layer.js]: Should this be `GIF Content`?
```suggestion
      {alt || __('GIF Content', 'web-stories')}
```

File: assets/src/edit-story/elements/image/layer.js [hunk 3]
```diff
@@ -13,25 +13,28 @@
...
-}) {
-  const src = getSmallestUrlForWidth(0, resource);
-  return <VisibleImage src={src} alt={alt} height="20" />;
-}
+function ImageLayerContent({ isBackground, element }) {
+  const { alt } = element?.resource || {};
 
+  return (
+    <LayerText isBackground={isBackground}>
+      {alt || __('Image', 'web-stories')}
```
Comment (samwhale) [inline assets/src/edit-story/elements/image/layer.js]: Should this be `Image Content`?
```suggestion
      {alt || __('Image Content', 'web-stories')}
```

File: assets/src/edit-story/elements/shape/layer.js [hunk 4]
```diff
@@ -13,87 +13,32 @@
...
-const ShapePreviewContainer = styled.div`
-  ${({ isTooBright }) =>
-    isTooBright &&
-    `
-      /* Using filter rather than box-shadow to correctly follow
-      * outlines of shapes.
-      */
-      filter: drop-shadow( 0 0 5px rgba(0, 0, 0, 0.5) );
-    `}
-`;
```
Comment (samwhale) [inline assets/src/edit-story/elements/shape/layer.js:56]: The designs didn't have these drop-shadows so I removed them from all of the logos. Do we still need these?

File: assets/src/edit-story/elements/shape/layer.js [hunk 5]
```diff
@@ -13,87 +13,32 @@
...
-              </clipPath>
-            </defs>
-          </svg>
-        </ShapePreview>
-      </ShapePreviewContainer>
-
-      {maskDef.name}
-    </Container>
+    <LayerText isBackground={isBackground}>
+      {maskDef.name || __('Shape', 'web-stories')}
```
Comment (samwhale) [inline assets/src/edit-story/elements/shape/layer.js]: I added a fallback here for the shape name. Most likely this will never be needed but if it is: should this be `Shape Content`?
```suggestion
      {maskDef.name || __('Shape Content', 'web-stories')}
```

File: assets/src/edit-story/elements/shape/layer.js (hunk 4 above)
Comment (barklund) [reply assets/src/edit-story/elements/shape/layer.js:56]: It's a remnant of a very old layer panel that had a white background, so any all-white shape wouldn't show up. I assume we would have the same problem today with a very dark shape though, but we can deal with that in a follow-up ticket, as that would be new functionality.

File: assets/src/edit-story/components/panels/design/layer/layer.js [hunk 6]
```diff
@@ -86,13 +81,15 @@ const LayerDescription = styled.div`
...
-  height: 18px !important;
-  width: 18px !important;
+const IconWrapper = styled.div`
+  position: absolute;
+  right: 18px;
 `;
 
-const BackgroundDescription = styled.div`
-  opacity: 0.5;
+const LockedIcon = styled(Locked)`
```
Comment (barklund) [inline assets/src/edit-story/components/panels/design/layer/layer.js]: This should be updated to use the icon from the design system, where it exists as `<Icons.LockClosed />`.

File: assets/src/edit-story/components/panels/design/layer/layer.js [hunk 7]
```diff
@@ -86,13 +81,15 @@ const LayerDescription = styled.div`
   font-size: ${({ theme }) => theme.DEPRECATED_THEME.fonts.description.size};
```
Comment (barklund) [inline assets/src/edit-story/components/panels/design/layer/layer.js]: All text fields must be updated to using the new typography system in the design system. There must be no direct references to font-sizes or anything like that anymore.

Also, no references to `DEPRECATED` colors either - that's a clear sign of something needing to be upgraded.

File: assets/src/edit-story/components/panels/design/layer/layer.js [hunk 8]
```diff
@@ -118,22 +114,16 @@ function Layer({ layer }) {
...
+        <LayerIcon element={layer} />
       </LayerIconWrapper>
       <LayerDescription>
-        {showPreview && (
-          <LayerContentContainer>
-            <LayerContent element={layer} />
-          </LayerContentContainer>
-        )}
+        <LayerContentContainer>
+          <LayerContent element={layer} isBackground={isBackground} />
```
Comment (barklund) [inline assets/src/edit-story/components/panels/design/layer/layer.js]: It seems wasteful to ask the individual layer for its content if `isBackground` is true - in that case, whatever the layer says will be ignored and replaced with `'Background'` anyway. Can't we just directly insert that here as `{isBackground ? <LayerText isBackground /> : <LayerContent element={layer}  />}` and then we don't have to prop-drill `isBackground`?

File: assets/src/edit-story/components/panels/design/layer/layer.js [hunk 9]
```diff
@@ -118,22 +114,16 @@ function Layer({ layer }) {
...
+        </LayerContentContainer>
         {isBackground && (
-          <BackgroundDescription>
-            {__('Background (locked)', 'web-stories')}
-          </BackgroundDescription>
+          <IconWrapper>
+            <LockedIcon aria-label={__('Background element', 'web-stories')} />
```
Comment (barklund) [inline assets/src/edit-story/components/panels/design/layer/layer.js]: This whole panel is `aria-hidden`, so labels and other `aria-*` properties are irrelevant. Non-mouse users have to use keyboard shortcuts to interact with the layers instead.

File: assets/src/edit-story/elements/shape/icon.js [hunk 10]
```diff
@@ -17,16 +17,64 @@
...
+  justify-content: center;
+  align-items: center;
+  height: 28px;
+  width: 28px;
+  border-radius: ${({ theme }) => theme.borders.radius.small};
+  background-color: ${({ theme }) => theme.colors.opacity.black10};
+`;
+
+const ShapePreview = styled.div`
+  ${elementWithBackgroundColor}
```
Comment (barklund) [inline assets/src/edit-story/elements/shape/icon.js:41]: Please check with @samitron7 if she wants shapes colored with their actual on-screen color (or gradient or whatever) or just always in a neutral white. The Figma design uses the same neutral white (`fg.primary`) as in the shape library, but that might just be a coincidence,

File: assets/src/edit-story/elements/shape/layer.js (hunk 5 above)
Comment (barklund) [reply assets/src/edit-story/elements/shape/layer.js]: `'Shape'` is fine, I think.

File: assets/src/edit-story/elements/shared/layerText.js [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { __ } from '@web-stories-wp/i18n';
+/**
+ * Internal dependencies
+ */
+import { Text, THEME_CONSTANTS } from '../../../design-system';
+
+export const LayerText = ({ isBackground, children, ...props }) => (
```
Comment (barklund) [inline assets/src/edit-story/elements/shared/layerText.js]: In general I prefer the `function LayerText() {}` over `const LayerText = () => {}` notation and I feel like remembering that it's recommended too, but can't find a link right now. I at least think we should try to be consistent.

🤔  I do think the former helps set `Component.displayName` for use in debugging contexts, as a "real" function is aware of its own (original) name, which an unnamed arrow function isn't. But that's probably the least relevant concern. However, if it's the only difference, it's probably enough to sway me 🤷 .

File: assets/src/edit-story/elements/text/icon.js [hunk 12]
```diff
@@ -18,15 +18,25 @@
...
+  width: 30px;
+`;
 
 function TextIcon() {
   const alt = __('Text element', 'web-stories');
-  return <Icon aria-label={alt} />;
+
+  return (
+    <IconContainer>
+      <Icons.LetterT aria-label={alt} />
```
Comment (barklund) [inline assets/src/edit-story/elements/text/icon.js]: Again, no label needed.

File: assets/src/edit-story/elements/media/visibleImage.js [hunk 13]
```diff
@@ -18,39 +18,16 @@
...
-    isTooBright &&
-    `
-    /* Using filter rather than box-shadow to correctly follow
-     * outlines in semi-transparent images like gif and png.
-     */
-    filter: drop-shadow( 0 0 5px rgba(0, 0, 0, 0.5) );
-  `}
+  height: 28px;
+  width: 28px;
+  border-radius: ${({ theme }) => theme.borders.radius.small};
```
Comment (barklund) [inline assets/src/edit-story/elements/media/visibleImage.js:26]: This image is stretched/squeezed to fit the square box. It should probably have a `object-fit: cover` to make it look better (but of course cropped).

File: assets/src/edit-story/elements/text/icon.js [hunk 14]
```diff
@@ -18,15 +18,25 @@
...
+import styled from 'styled-components';
 /**
  * Internal dependencies
  */
-import { ReactComponent as Icon } from './icon.svg';
+import { Icons } from '../../../design-system';
+
+const IconContainer = styled.div`
+  height: auto;
+  width: 30px;
```
Comment (barklund) [inline assets/src/edit-story/elements/text/icon.js]: Isn't it 32px?

Screen Shot 2021-03-02 at 21 09 29

**Review by barklund — CHANGES_REQUESTED** [review]
I'm generally missing edits for the `layer/layerPanel.js` file here - where's all the styling of the overall layer panel? It's significantly different from the other panels, slightly narrower, has rounded edges, a shorter divider, a different background color (`bg.tertiary` rather than `bg.secondary`, and the whole panel has a drop-shadow. I can only find it [in the context of the whole design panel in Figma in the collapsed state](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=221%3A13978), but it should look the same with [the expanded state](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3360%3A63235):

_Collapsed with context_:
Screen Shot 2021-03-02 at 20 43 22
_Expanded state without context_:
Screen Shot 2021-03-02 at 20 46 38


There's also a todo and screenshots mentioning all this in the parent ticket in #6210, that doesn't seem to have been touched.

File: assets/src/edit-story/elements/gif/layer.js (hunk 2 above)
Comment (spacedmonkey) [reply assets/src/edit-story/elements/gif/layer.js:32]: How do you know the image is a gif here?

**Comment by samwhale** [comment]
thanks for the thorough review @barklund. I missed a lot of stuff so thanks for catching all of that.

I updated the layer panel with all of your recommended fixes and added the missing styling that the ticket was asking for:


Note: the scrollbar looks weird in the gif but I promise it doesn't like that in the actual browser. However, I think some more work needs to be done to make the scrollbar and the divider look nice: https://xwp.slack.com/archives/C01ESBB8ZN1/p1614791411042500

~I'm still waiting on some feedback from @samitron7, but this is much closer to what we wanted.~ feedback implemented

File: assets/src/edit-story/components/panels/design/layer/layer.js [hunk 15]
```diff
@@ -109,7 +103,6 @@ function Layer({ layer }) {
     currentPage: state.state.currentPage,
   }));
   const isBackground = currentPage.elements[0].id === layer.id;
-  const showPreview = !isBackground || layer.type !== 'shape';
 
   return (
     <LayerButton
```
Comment (samwhale) [inline assets/src/edit-story/components/panels/design/layer/layer.js:115]: just chatted with Sam about the button states. She said these should follow the same rules as what we use for rectangle buttons.

I'll update this!

File: assets/src/edit-story/elements/shape/icon.js (hunk 10 above)
Comment (samwhale) [reply assets/src/edit-story/elements/shape/icon.js:41]: Chatted with Sam - she said to keep the shapes as close to what the color is on the page if possible

**Review by embarks — APPROVED** [review]
Checked it out on local, looks great! ✨

File: assets/src/edit-story/components/panels/design/layer/constants.js [hunk 16]
```diff
@@ -15,4 +15,4 @@
  */
 
 export const DEFAULT_LAYERS_VISIBLE = 6;
-export const LAYER_HEIGHT = 40;
+export const LAYER_HEIGHT = 44;
```
Comment (BrittanyIRL) [inline assets/src/edit-story/components/panels/design/layer/constants.js:18]: I'm seeing 48 in figma for layer height?
Screen Shot 2021-03-04 at 1 24 09 PM

File: assets/src/edit-story/components/panels/design/layer/layerPanel.js [hunk 17]
```diff
@@ -27,31 +28,56 @@ import { LAYER_HEIGHT, DEFAULT_LAYERS_VISIBLE } from './constants';
 import LayerList from './layerList';
 import useLayers from './useLayers';
 
+const Container = styled.div`
+  margin: 0 4px;
+  filter: drop-shadow(0px -4px 8px rgba(0, 0, 0, 0.2));
```
Comment (BrittanyIRL) [inline assets/src/edit-story/components/panels/design/layer/layerPanel.js:33]: Does this duplicate? Would it be good to store in the theme? Or at least pull the rgba in as a theme color added?

**Review by BrittanyIRL — APPROVED** [review]
Looks good across firefox, chrome, safari (both LTR and RTL 😉 )! 

I'm pretty sure this is out of scope, but the drag n drop color is still blue, wanted to confirm this is getting done elsewhere. 
blue drag and drop

Same here, this is an enhancement but I wanted to mention it - It'd be cool if the fallback label for images is the assistive text. Pulled this in from unsplash - so instead of "image" it would be "a sunset" 
Screen Shot 2021-03-04 at 1 36 44 PM

File: assets/src/edit-story/components/panels/design/layer/constants.js (hunk 16 above)
Comment (samwhale) [reply assets/src/edit-story/components/panels/design/layer/constants.js:18]: Oh nice catch. Looks like I missed that

The layers are 44px so I'll need to change just the height of just the header instead of the layer height (which is used by all the buttons as well)

**Comment by samwhale** [comment]
> I'm pretty sure this is out of scope, but the drag n drop color is still blue, wanted to confirm this is getting done elsewhere.

Made the button active color consistent

> Same here, this is an enhancement but I wanted to mention it - It'd be cool if the fallback label for images is the assistive text. Pulled this in from unsplash - so instead of "image" it would be "a sunset"

Added the display name to the layer panel for images

**Review by maxyinger — APPROVED** [review]
Noice work!

File: assets/src/edit-story/components/panels/design/layer/layerPanel.js (hunk 17 above)
Comment (samwhale) [reply assets/src/edit-story/components/panels/design/layer/layerPanel.js:33]: Looks like this same value is used here: https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/elements/video/controls.js#L85-L91

This was an old change so I don't know if it will survive. I don't see the color anywhere else, so I'd say we leave it for now 🤷

Comment (BrittanyIRL) [reply assets/src/edit-story/components/panels/design/layer/layerPanel.js:33]: works for me!

File: assets/src/edit-story/elements/image/layer.js [hunk 18]
```diff
@@ -13,24 +13,27 @@
...
-  const src = getSmallestUrlForWidth(0, resource);
-  return <VisibleImage src={src} alt={alt} height="20" />;
-}
+function ImageLayerContent({ element, ...props }) {
+  const { alt, attribution } = element?.resource || {};
+  const displayName = attribution?.author?.displayName;
 
+  return (
+    <LayerText {...props}>
+      {displayName || alt || __('Image', 'web-stories')}
```
Comment (barklund) [inline assets/src/edit-story/elements/image/layer.js]: Not sure I editorially agree with this - alt should be the primary label I think. And the attribution author doesn't seem relevant to me. If I update the image alt in the accessibility panel, this label should update (it doesn't currently for neither local media nor unsplash media).

But that's a nit.

File: assets/src/edit-story/components/panels/design/layer/layerPanel.js [hunk 19]
```diff
@@ -27,31 +28,57 @@ import { LAYER_HEIGHT, DEFAULT_LAYERS_VISIBLE } from './constants';
...
+  border-top-right-radius: ${({ theme }) => theme.borders.radius.small};
+`;
+
+const DividerContainer = styled.div`
+  background: ${({ theme }) => theme.colors.bg.tertiary};
+`;
+
+const Divider = styled.div`
+  border-bottom: 1px solid ${({ theme }) => theme.colors.divider.tertiary};
+  margin: 0 16.5px;
```
Comment (barklund) [inline assets/src/edit-story/components/panels/design/layer/layerPanel.js:48]: Why 16.5? It looks to be 8.5 in [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3360%3A63235):

Screen Shot 2021-03-04 at 17 27 17

And it should probably actually be 8px so it aligns with the layer icons below, which are (supposed to be) 8px from the panel edge.

File: assets/src/edit-story/components/panels/design/layer/layer.js [hunk 20]
```diff
@@ -46,53 +47,47 @@ const LayerButton = styled.button.attrs({
   overflow: hidden;
   align-items: center;
   user-select: none;
+  border-radius: 0;
```
Comment (barklund) [inline assets/src/edit-story/components/panels/design/layer/layer.js:51]: Should this one have `justify-content: flex-start`? Currently it `space-around`'s the elements (inherited from the general button), which adds a bit of extra space to the left of the icon, which should be 8px from the edge, but is more like 10px off.

See this screenshot, where I manually added `justify-content: flex-start` to the selected layer in devtools and notice how the icon is "correctly" located 8px from the edge.

Screen Shot 2021-03-04 at 17 37 02

File: assets/src/edit-story/components/panels/design/layer/layer.js [hunk 21]
```diff
@@ -46,53 +47,47 @@ const LayerButton = styled.button.attrs({
...
+
+  :active {
+    background: ${({ theme }) => theme.colors.interactiveBg.secondaryPress};
+  }
```
Comment (barklund) [inline assets/src/edit-story/components/panels/design/layer/layer.js:60]: Nit: should layers have a nice hover background? Maybe `interactiveBg.secondaryHover` to go with the selected state?

Every nittier: I personally feel the transition time is a little long, but I get that it comes from the general button. `.6s` is a lot IMHO, I think `.3s` looks better, but that's purely opinion, so feel free to ignore me.

File: assets/src/edit-story/elements/gif/layer.js [hunk 22]
```diff
@@ -13,24 +13,22 @@
...
-function GifLayerContent({
-  element: {
-    resource,
-    resource: { alt },
-  },
-}) {
-  const src = getSmallestUrlForWidth(0, resource);
-  return <VisibleImage src={src} alt={alt} height="20" />;
-}
+function GifLayerContent({ element, ...props }) {
```
Comment (barklund) [inline assets/src/edit-story/elements/gif/layer.js]: What would `...props` be here (and in all the other layer contents)? I only see this used[ in a single place](https://github.com/google/web-stories-wp/pull/6590/files#diff-0cbdd980ae4ab69502a003f978d41ed8ede38e7dc613a0204513a04e451e592fR122), and it passes no other properties. Or is it used elsewhere?

File: assets/src/edit-story/elements/shared/layerText.js [hunk 23]
```diff
@@ -0,0 +1,46 @@
...
+
+const StyledText = styled(Text)`
+  color: inherit;
+  white-space: nowrap;
+  text-overflow: ' ';
+  overflow: hidden;
+  max-width: 100%;
+`;
+
+export function LayerText({ children, ...props }) {
```
Comment (barklund) [inline assets/src/edit-story/elements/shared/layerText.js]: Similarly here - does it ever get `...props`?

**Review by barklund — APPROVED** [review]
This is great work - just a couple of small nits and a few layout details.

File: assets/src/edit-story/components/panels/design/layer/layerPanel.js (hunk 19 above)
Comment (samwhale) [reply assets/src/edit-story/components/panels/design/layer/layerPanel.js:48]: Sam said we should increase the spacing around the divider because it looked like this when the scrollbar appeared:


I can change it back too but it needed a little bit of space. Should we change it back to 8px?

Chatted with Sam about this here: https://xwp.slack.com/archives/C01ESBB8ZN1/p1614791411042500

File: assets/src/edit-story/elements/image/layer.js (hunk 18 above)
Comment (samwhale) [reply assets/src/edit-story/elements/image/layer.js]: I'll remove it!

File: assets/src/edit-story/elements/gif/layer.js (hunk 22 above)
Comment (samwhale) [reply assets/src/edit-story/elements/gif/layer.js]: I did this just in case we needed to pass extra props to the text element, but you're right - it isn't needed atm. I can remove this stuff.

File: assets/src/edit-story/components/panels/design/layer/layerPanel.js (hunk 19 above)
Comment (BrittanyIRL) [reply assets/src/edit-story/components/panels/design/layer/layerPanel.js:48]: re the slack thread Samuel posted, @samitron7 's response was `Just adding more padding between them would work.`

File: assets/src/edit-story/components/panels/design/layer/layer.js (hunk 21 above)
Comment (BrittanyIRL) [reply assets/src/edit-story/components/panels/design/layer/layer.js:60]: Re the button transition: the button transition _does_ feel long. I just grabbed that from the existing buttons we had in the dashboard, extremely arbitrary. Let's change it in a separate PR!

File: assets/src/edit-story/components/panels/design/layer/layerPanel.js (hunk 19 above)
Comment (barklund) [reply assets/src/edit-story/components/panels/design/layer/layerPanel.js:48]: Fine with me! Though half-pixels always bug me 🤷‍♂️

File: assets/src/edit-story/components/panels/design/layer/constants.js (hunk 16 above)
Comment (BrittanyIRL) [reply assets/src/edit-story/components/panels/design/layer/constants.js:18]: Thanks for updating!

File: assets/src/edit-story/components/panels/design/layer/layer.js (hunk 20 above)
Comment (BrittanyIRL) [reply assets/src/edit-story/components/panels/design/layer/layer.js:51]: Screen Shot 2021-03-05 at 10 03 52 AM

confirmed, alignment is fixed

File: assets/src/edit-story/components/panels/design/layer/layer.js (hunk 21 above)
Comment (BrittanyIRL) [reply assets/src/edit-story/components/panels/design/layer/layer.js:60]: Hover's updated: 
updated hover bg

ticket for button transition: https://github.com/google/web-stories-wp/issues/6635
